CityOfDetroit/bloom
# fix: passing tz as param

# Pull Request Template

## Issue Overview

This PR addresses #1606 


## Description

This PR updates the listings export and csv export to show times in the user's timezone rather than in UTC to improve readability.

## How Can This Be Tested/Reviewed?

This can be tested by pulling down the branch locally, reseeding, and then export the listing zip. In the listing csv, you will see the created at date will be the local time that you reseeded along with the appropriate timezone abbreviation. To test the other dates, you can update a listing, verify a listing, etc.. Then save and export to ensure the times show appropriately.

Then, follow the same general testing approach for the user export.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
